pkg/commands/workload.go

```diff
@@ -792,13 +794,35 @@ func (opts *WorkloadOptions) GetTailWorkers(c *cli.Config, workload *cartov1alph
 	return workers
 }
 
+func getClusterSupplyChainTypeSelectors(fields []metav1.LabelSelectorRequirement) []string {
+	var values []string
+	for _, v := range fields {
+		if v.Key == apis.WorkloadTypeLabelName && len(v.Values) != 0 {
```

Comment:
`len` check is not required, see https://go.dev/play/p/7tJe1k_QO_v

Reply:
ok, removed `len` validation